core/src/main/scala/org/apache/spark/deploy/security/HadoopDelegationTokenManager.scala

```diff
@@ -55,7 +56,9 @@ private[spark] class HadoopDelegationTokenManager(
   logDebug(s"Using the following delegation token providers: " +
     s"${delegationTokenProviders.keys.mkString(", ")}.")
 
-  /** Construct a [[HadoopDelegationTokenManager]] for the default Hadoop filesystem */
+  /** Construct a [[HadoopDelegationTokenManager]] for the default Hadoop filesystem with a
```

Comment:
Comment starts on next line. But the comment doesn't really seem accurate anyway.